main.py

```diff
@@ -60,10 +67,16 @@ def main():
     data_cons.start()
     log_cons.start()
 
-    # Start the GUI. The `run` method contains a loop which will keep the program running
-    print("STARTING GUI...")
-    gui.run()
+    # Shutdown: we wait for a button to be pressed before triggering the shutdown procedure
+    print("GUI disabled in development")
+    while not start_button.was_pressed():
+        time.sleep(0.2) # give priority to other threads
 
+    stopper.normal_shutdown()
+
+    # Start the GUI. The `run` method contains a loop which will keep the program running
+    # print("STARTING GUI...")
```

Comment:
remove comments unless you feel they are really worth keeping

Reply:
en developpement si c'est pour se rappeler de certains trucs c'est chill

Reply:
Je vais tout regader ça quand on sera prêt à intégrer à nouveau l'interface. Voir https://github.com/Exocet-Polytechnique/ExoPIBrain/issues/4.